src/main/java/com/revature/cmsforce/CMSforceApplication.java

```diff
@@ -25,8 +25,11 @@
 	public static void main(String[] args) {
 		SpringApplication.run(CMSforceApplication.class, args);
 		
-	
 		
 	}
 
 }
+
+interface Test{
```

Comment:
What is this interface for?

Reply:
I think that was a simple overlook. We will remove it.